### ISSUE TITLE
zerocracy should have a StackOverflow tag

### ISSUE THREAD

**Issue by amihaiemil** [zerocracy/datum#316]
Zerocracy should be present on StackOverflow, since a lot of users will have questions and many of them won't take the time to study all the open and closed Issues in ZC's repos. Besides, Issues are more for bug reports than for questions.

**Comment by 0crat** [comment]
@yegor256[/z](https://www.0crat.com/u/yegor256) please, pay attention to this issue

**Comment by amihaiemil** [comment]
@yegor256 I think you should take care of this :)

**Comment by yegor256** [comment]
@amihaiemil I don't think they will accept that, since it's not a software product or a technology. Maybe pm.stackexchange.com?

**Comment by amihaiemil** [comment]
@yegor256 that could also work. Basically, questions, unclarities etc, should be settled there, rather than a flood of tickets :)

**Comment by yegor256** [comment]
@amihaiemil ok, we will do that, once Zerocracy is launched.

**Comment by yegor256** [comment]
@amihaiemil maybe we can create zerocracy.stackexchange.com?

**Comment by yegor256** [comment]
@amihaiemil or maybe xdsd.stackexchange.com?

**Comment by amihaiemil** [comment]
@yegor256 zerocracy.stackexchange.com is better. Zerocracy is just an implementation of XDSD and PDD. There may be more in the future, competitors :P

**Comment by yegor256** [comment]
@amihaiemil yes, we will do it.

**Comment by ruspl-afed** [comment]
@yegor256  still no progress here
StackOverflow is "to help you with specific coding, algorithm, or language problems" - seems to be offtopic for the most part of zerocracy-related questions
But some resource is needed with a link to it from https://www.0crat.com/

**Comment by yegor256** [comment]
@ruspl-afed definitely, we should think about this. In the meantime, there is Telegram group to help you solve most of the problems: https://t.me/zerocracy
